src/lib/config.ts

```diff
@@ -97,6 +97,8 @@ export const REGISTRY2_HOST = requiredVar('REGISTRY2_HOST');
 export const REGISTRY_HOST = requiredVar('REGISTRY_HOST');
 export const SENTRY_DSN = process.env.SENTRY_DSN;
 export const SUPERUSER_EMAIL = process.env.SUPERUSER_EMAIL || '';
+export const SUPERUSER_USERNAME =
+	process.env.SUPERUSER_EMAIL || SUPERUSER_EMAIL.split('@')[0];
```

Comment:
I'm not sure where the requirement for this has come from, but I think we should keep using `root` as the default username.